src/main/java/nl/thehyve/ocdu/factories/PatientDataFactory.java

```diff
@@ -82,4 +81,30 @@ private void setValue(String[] row, Map<String, Integer> columnsIndex, String co
     }
 
 
+    public List<String> generatePatientRegistrationTemplate(MetaData metadata, Map<String, String> subjectMap, boolean registerSite) {
+        List<String> result = new ArrayList<>();
+        String header = "Study Subject ID\t";
```

Comment:
String.join is better suited when the elements are not conditioned, whereas in this case we have some elements (e.g. gender and date of birth) that could or could not be joined in the final string, so using the simple "+" sign is simpler